app/models/data.py

```diff
@@ -96,26 +135,35 @@ class CoreData(db.Model, DataMixin):
     # What other columns are we missing?
     sourceNotes = db.Column(db.String)
 
-    def __init__(self, **kwargs):
-        super(CoreData, self).__init__(**kwargs)
-
     @hybrid_property
     def lastUpdateEt(self):
         # convert lastUpdateTime (UTC) to ET
-        raise NotImplementedError
+        return self.lastUpdateTime.replace(tzinfo=pytz.utc).astimezone(pytz.timezone('US/Eastern'))
 
     @hybrid_property
     def checkTimeEt(self):
         # convert lastCheckTime (UTC) to ET
-        raise NotImplementedError
+        return self.lastCheckTime.replace(tzinfo=pytz.utc).astimezone(pytz.timezone('US/Eastern'))
 
     @hybrid_property
     def totalTestResults(self):
         # Calculated value (positive + negative) of total test results.
-        raise NotImplementedError
+        if self.positive is None or self.negative is None:
+            return None
```

Comment:
For consistency with the current public API, I think we need to treat nulls as 0 for this purpose. I checked PR, which does not report negatives, and the public API just has totalTestResults=positives.